Lib/test/test_asyncio/functional.py

```diff
@@ -60,21 +60,13 @@ def tcp_server(self, server_prog, *,
             else:
                 addr = ('127.0.0.1', 0)
 
-        sock = socket.socket(family, socket.SOCK_STREAM)
-
+        sock = socket.bind_socket(addr, family=family, backlog=backlog)
```

Comment:
Previously, the timeout was set before bind+listen. Now it's set after. Do you know if bind or listen are impacted by the timeout? (I don't think so.)

Reply:
No. Same effect. `accept()` is the one affected by the timeout but setting it before or after is the same.